build/dependencies.yaml

```diff
@@ -132,7 +132,7 @@ dependencies:
 
   # Base images
   - name: "k8s.gcr.io/debian-base: dependents"
-    version: buster-v1.4.0
+    version: buster-v1.5.0
```

Comment:
I think we've actually just built a new one last week, debian-base is up to buster-v1.6.0 now.